pipelines/build/common/build_base_file.groovy

```diff
@@ -61,23 +61,17 @@ class Builder implements Serializable {
         'sanity.system',
         'extended.system',
         'sanity.perf',
-        'sanity.external'
+        'sanity.functional',
+        'extended.functional'
     ]
-    
-    // Temporarily remove weekly tests due to lack of machine capacity during release
-    final List<String> weekly = []
-    /*
+
     final List<String> weekly = [
```

Comment:
Can we get a short explanation or link about the test selection? Will daily and weekly be combined during release? Will both run on a weekend?

Reply:
I've pushed some comments - and will update the "Graduated Testing wiki, once we actually schedule a 1x/week weekly build pipeline

Reply:
Thanks!